pyomo/contrib/piecewise/transform/incremental.py

```diff
@@ -0,0 +1,203 @@
+#  ___________________________________________________________________________
+#
+#  Pyomo: Python Optimization Modeling Objects
+#  Copyright (c) 2008-2024
+#  National Technology and Engineering Solutions of Sandia, LLC
+#  Under the terms of Contract DE-NA0003525 with National Technology and
+#  Engineering Solutions of Sandia, LLC, the U.S. Government retains certain
+#  rights in this software.
+#  This software is distributed under the 3-clause BSD License.
+#  ___________________________________________________________________________
+
+from pyomo.contrib.fbbt.fbbt import compute_bounds_on_expr
+from pyomo.contrib.piecewise.transform.piecewise_linear_transformation_base import (
+    PiecewiseLinearTransformationBase,
+)
+from pyomo.contrib.piecewise.triangulations import Triangulation
+from pyomo.core import (
+    Constraint,
+    Binary,
+    NonNegativeIntegers,
+    Suffix,
+    Var,
+    RangeSet,
+    Param,
+)
+from pyomo.core.base import TransformationFactory
+from pyomo.gdp import Disjunct, Disjunction
+from pyomo.common.errors import DeveloperError
+from pyomo.core.expr.visitor import SimpleExpressionVisitor
```

Comment:
I don't think you need this import.

Reply:
You are correct. What was I even doing with this?